src/Docker/Runner.php

```diff
@@ -240,6 +240,7 @@ private function runRow(JobDefinition $jobDefinition, $action, $mode, $jobId, Us
                 $this->loggersService->getLog(),
                 $workingDirectory->getDataDir(),
                 $configData['storage'],
+                $configData['runtime'],
```

Comment:
I'd put in the entire $jobDefinition, the refactoring will be easier and the interface cleaner (in future when the redundant fields are removed)